openage/convert/interface/visgrep.pyx

```diff
@@ -15,6 +15,7 @@ cimport numpy
 
 from collections import namedtuple
 from PIL import Image
+from sys import exit
```

Comment:
I don't think this is needed?
https://docs.python.org/3/library/constants.html#exit

Reply:
A couple lines above the linked section, it says "... They are useful for the interactive interpreter shell and should not be used in programs."
I'd say openage qualifies as a program (at least). I hope this change is acceptable. Otherwise, I can (`import sys` here, and) replace the usages of `exit` with `sys.exit` inline to make it work.

Whatever the decision, I'll make the same change to `__main__.py` as it faces the same problem (mentioned in my response to the previous comment).